tests/integration_tests/build/test_clippy.py

```diff
@@ -22,6 +23,8 @@ def test_rust_clippy(target):
 
     @type: build
     """
-    utils.run_cmd(
-        "cargo clippy --target {} --all --profile test" " -- -D warnings".format(target)
-    )
+    cmd = "cargo clippy --target {} --all --profile test" " --".format(target)
```

Comment:
nit: make this a single string

also: why do we have to append the "-D warnings" below, can't we include the it here, like it was before?

Reply:
What do you mean I should make a single string?